### PR TITLE
Move client command documentation to client

### PR DESCRIPTION
**Pull request by asgdf**
As per https://github.com/Zarel/Pokemon-Showdown-Client/pull/280#issuecomment-65842063

### CODE REVIEW

File: js/client-chat.js [hunk 1]
```diff
@@ -684,6 +692,45 @@
 					Tools.prefs('avatar', avatar);
 				}
 				return text; // Send the /avatar command through to the server.
+				
+			// documentation of client commands
+			case 'help':
+				switch (target) {
```
Comment (Zarel) [inline js/client-chat.js]: `(target||'').toLowerCase()`?`

Comment (Zarel) [inline js/client-chat.js]: idr if our commands are case-sensitive

Comment (Zarel) [inline js/client-chat.js]: Apparently they're not

**Comment by asgdf** [comment]
Ah right, forgot case-sensitivity. Fixed that.

**Pull request merged**